yb-voyager/src/migassessment/sizing.go

```diff
@@ -501,10 +560,9 @@ func checkFileExistsOnRemoteRepo(fileName string) bool {
 	}(resp.Body)
 
 	if resp.StatusCode != 200 {
-		//fmt.Println("File does not exist on remote location")
 		return false
 	} else {
-		downloadPath := strings.ReplaceAll(fileName, "resources/", baseDownloadPath)
+		downloadPath := assessmentMetadataDir + fileName
 		out, _ := os.Create(downloadPath)
```

Comment:
don't ignore the error returned by `os.Create` here, or any function, irrespective of how unlikely it is to occur.

Reply:
handled the error